components/browser/search/src/main/assets/searchplugins/daum-kr.xml

```diff
@@ -6,13 +6,13 @@
 <ShortName>다음지도</ShortName>
 <InputEncoding>utf-8</InputEncoding>
 <Image width="16" height="16">data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGAAAABgCAMAAADVRocKAAAARVBMVEUAAAD5+fn5+fn5+fn5+fn5+fn5+fn5+fn5+fn5+fn5+flNkP//1Smn3wAoffz9dG7kWlfjngNhtAAke/0gdv9isiRZrCSQTyACAAAACnRSTlMAwwbS+W9sG8/zKHP4TgAAALhJREFUaN7t2ksOwjAMRdGXfzEQoAH2v1QKE6ARIi0yQvTdmSc5Y0cGYKPxopA30WIodaJWlwDrRDFnEUW1CCOqGXhRzUOUI0CAAAECBL4D5MaO+dyPO1TtqyYA/bKBcu3+Wrn1NM0HmjvNB0peN7St2lTtHiJAgAABAgQIECBAgAABApOB1j5cY9/264s4gdfAH/x4jWYCBAiIECCwSMCLaiv9s4cgqgX90xMkRcElDNlgRCUTLHAB68OktAP7gdIAAAAASUVORK5CYII=</Image>
-<Url type="application/x-suggestions+json" method="GET" template="http://sug.search.daum.net/search_nsuggest">
+<Url type="application/x-suggestions+json" method="GET" template="https://sug.search.daum.net/search_nsuggest">
```

Comment:
This one I can't resolve, but that's true for the `http` version too.

Reply:
I was a bit unsure about how to resolve the issue should I remove it or not.
Ps: 1st PR on android-components sorry about that